packages/ui/src/styles/base/layout.scss

```diff
@@ -75,3 +75,13 @@ body {
   height: 100%;
   object-fit: cover;
 }
+
+////////////////////
+// Pages
```

Comment:
Oops I think I missed it on cherry pick 🙈

Reply:
@icazevedo can you remove this file?